# SetPageColumn

std_msgs/Header header
uint16 page
uint16 column
